fe1-web/__tests__/model/objects/Objects.test.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  Base64Data,
+  Hash, KeyPair, KeyPairState,
+  Lao, LaoState,
+  PrivateKey,
+  PublicKey,
+  WitnessSignature,
+} from 'model/objects';
+import { channelFromId, ROOT_CHANNEL } from 'model/objects/Channel';
+
+const mockPublicKey = new PublicKey('xjHAz+d0udy1XfHp5qugskWJVEGZETN/8DV3+ccOFSs=');
+const mockSecretKey = new PrivateKey('vx0b2hbxwPBQzfPu9NdlCcYmuFjhUFuIUDx6doHRCM7GMcDP53S53LVd8enmq6CyRYlUQZkRM3/wNXf5xw4VKw==');
```

Comment:
Should we then import those keys from the `tests/data/keypair.json` file?

(Same comment for the other test files using the keys)

Reply:
Yes, I just haven't gotten around to really-really cleaning up the other non-MessageApi.test.ts files :)